### PR TITLE
docs: update how-to-build.md

### PR DESCRIPTION
**Pull request by candyleer**
add command make install in case of `command not found`

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/how-to-build.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ Please refer to: [Installing Apache APISIX with Helm Chart](https://github.com/a
   cd apisix-2.9
   # Create dependencies
   make deps
+  # Install apisix command
+  make install
```
Comment (leslie-tsang) [inline docs/en/latest/how-to-build.md:80]: Please update [CN version](https://github.com/apache/apisix/blob/master/docs/zh/latest/how-to-build.md) at the same time.

**Review by leslie-tsang — CHANGES_REQUESTED** [review]

**Comment by tzssangglass** [comment]
why we change `make *` to `apisix *`? for example,  if `apisix` is not in the path search variable, you will get error as `command not found`.

`make stop` = `./bin/apisix stop`, it works well.

**Comment by candyleer** [comment]
> why we change `make *` to `apisix *`? for example, if `apisix` is not in the path search variable, you will get error as `command not found`.
> 
> `make stop` = `./bin/apisix stop`, it works well.

english version has been changed in https://github.com/apache/apisix/pull/5075, so chinese version follow up

**Comment by tzssangglass** [comment]
> > why we change `make *` to `apisix *`? for example, if `apisix` is not in the path search variable, you will get error as `command not found`.
> > `make stop` = `./bin/apisix stop`, it works well.
> 
> english version has been changed in #5075, so chinese version follow up

got

File: docs/en/latest/how-to-build.md (hunk 1 above)
Comment (tzssangglass) [reply docs/en/latest/how-to-build.md:80]: I think `make install` means installing the APISIX code into luarocks. The context here is to download the APISIX source code, so does the source code still need to be installed to luarocks? I thought it was no longer needed. what's your option? @spacewander @membphis

Comment (spacewander) [reply docs/en/latest/how-to-build.md:80]: Yes. See the comment here:
https://github.com/apache/apisix/blob/6162ca0ef92445fdd15b7a213655887b66cd4eea/Makefile#L167

Comment (spacewander) [reply docs/en/latest/how-to-build.md:80]: I changed my mind. Look like we still need this command to install APISIX to PATH even `make install` is designed for luarocks.

Comment (tzssangglass) [reply docs/en/latest/how-to-build.md:80]: why

Comment (spacewander) [reply docs/en/latest/how-to-build.md:80]: As there is no other command to do it.

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**